canopy.py

```diff
@@ -361,7 +411,66 @@ def correct_inverted_canopy_tiff(inverted_phyreg_ids):
                 corrected = 1 - arcpy.Raster(canopytif_path)
                 # copy raster is used as arcpy.save does not give bit options.
                 arcpy.CopyRaster_management(corrected, corrected_path,
-                        pixel_type='2_BIT')
+                                            nodata_value = '3',
+                                            pixel_type='2_BIT')
+
+    # clear selection
+    arcpy.SelectLayerByAttribute_management(phyregs_layer, 'CLEAR_SELECTION')
+
+    print('Completed')
+
+def convert_canopy_tif_to_shp(phyreg_id):
```

Comment:
Let's be consistent. All `tiff` or `tif`. Other functions use `tiff`.

1. `tiff_to_shapefile`: TIFF and Shapefile both are full names
2. `tiff_to_shp`: Only TIFF is a full name
3. `tif_to_shapefile`: Only Shapefile is a full name
4. `tif_to_shp`: Both are file extensions.

Either 1 or 4, but I think 4 looks better? Then, let's rename `convert_afe_to_canopy_tiff` and `correct_inverted_canopy_tiff` to ...`tif`.

Reply:
I agree #4 looks better and keeps the name shorter.